### PR TITLE
support xla_hlo.neg convert to vmla.neg

### PR DESCRIPTION
**Pull request by qqsun8819**
also with a test which test neg with vmla and llvm-ir backend

### CODE REVIEW

File: iree/test/e2e/xla/neg.mlir [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  return %result : tensor<4xf32>
+}
+// CHECK: 4xf32=1 2 -3 -4
+
+// -----
+
+// CHECK-LABEL: EXEC @scalar
+func @scalar() -> tensor<f32> {
+  %input = iree.unfoldable_constant dense<-4.0> : tensor<f32>
+  %result = "xla_hlo.abs"(%input) : (tensor<f32>) -> tensor<f32>
```
Comment (ScottTodd) [inline iree/test/e2e/xla/neg.mlir:18]: This should be `"xla_hlo.neg"` also

Comment (qqsun8819) [reply iree/test/e2e/xla/neg.mlir]: fixed

**Comment by qqsun8819** [comment]
> Ah, this op was just renamed in XLA: [tensorflow/tensorflow@be22aba](https://github.com/tensorflow/tensorflow/commit/be22aba837a270c4ec8e9ed5009958a30fe967e2) (from `xla_hlo.neg` to `xla_hlo.negate`). Can you rebase off of master and rename? Force pushing over this existing branch is fine (or you could merge with master and just add new commits... but rebase + force push is simpler IMO).

@ScottTodd ok , I'll update it

**Comment by qqsun8819** [comment]
@ScottTodd  pr updated, take a look again ^_^

**Pull request closed without merging**